.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,137 @@
+  feature requests in the issue tracker, which is a good place to start.
+
+* That the feature is in scope of the project
+
+  Some examples of features that are not in scope:
+
+    * Contact management, fetching email, sending email, indexing email.
+      Alot relies on external services for all of these features. Integration
+      with a new external service is okay.
+
+* If the feature is requested already, please use the thumbs up emoji to add
+  your support for the feature. Please do not add a "me too" comment.
+
+Once you're sure that the feature isn't implemented or already requested, add a
+request. You will be given a template to fill out, but don't, that template is
+for bugs. Please be as thorough as possible when describing the feature, please
+explain what it does, how it should work, and how you plan to use the feature;
+if applicable.
+
+
+Contributing
+------------
+
+
+Before you Contribute
+`````````````````````
+
+Alot is licensed under the `GNU GPLv3+
+<https://www.gnu.org/licenses/gpl-3.0.en.html>`_, and all code contributions
+will be covered by this license. You will retain your copyright on any code you
+write.
+
+By contributing you have agreed that you have the right to contribute the code
+you have written, that the code you are contributing is owned by you or
+licensed with a compatible license allowing it to be added to alot, and that
+your employer or educational institution either does not or cannot claim the
+copyright on any code you writhe, or that they have given you a waiver to
+contribute to alot.
+
+
+What to Contribute
+``````````````````
+
+* Bug fixes are always welcome
+
+* Tests for bugs. If you can replicate a bug but don't want to or can't fix it
+  we'll still take a unit test. Please decorate the test as an expected
+  failure if it is failing.
+
+* New features. Please be aware that we won't take every feature proposed,
+  especially those that expand the scope of the project.
+
+* Documentation. Including typos, spelling and grammar, wrong type and
+  parameter annotations, or documentation that has gotten out of sync with the
+  code.
+
+
+Sending a Pull Request
+``````````````````````
+
+The best way to send new code is with a pull request using the github interface.
+
+* Follow :pep:`8`. This means in particular a maximum linewidth of *79* and no trailing
+  white spaces. If in doubt, use an Automatic tool
+  (`[0] <http://www.logilab.org/857>`_, `[1] <http://pypi.python.org/pypi/pep8/>`_, `[2]
+  <http://pypi.python.org/pypi/pyflakes/>`_)
+  to verify your code.
+
+* Document! Needless to say, we want readable and well documented code. Moreover,
+
+  * use `sphinx directives <http://sphinx.pocoo.org/domains.html#info-field-lists>`_ to document
+    the parameters and return values of your methods so that we maintain up-to-date API docs.
+  * If you implemented a new feature, update the user manual in :file:`/docs/user` accordingly.
```

Comment:
I guess. That was in the docs from `docs/source/api/contributing.rst`, which I (at pazz's suggestion) re-purposed here.